style.css

```diff
@@ -0,0 +1,76 @@
+@import url("https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@400;700&display=swap");
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  background-color: rgb(244, 244, 244);
+  font-family: "Josefin Sans", sans-serif;
+
+}
+a{
+  text-decoration: none;
+  color: black;
+  font-size: 18px;
+}
+#navbar{
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  height: 50px;
+  align-items: center;
+  background-color: rgb(244, 243, 243);
+  position: sticky;
+}
+#welcome-section{
+  padding: 30px;
+  margin-top: 1rem;
+}
+#welcome-section h1 {
+  font-size: 32px;
+  font-family: "Josefin Sans", sans-serif;
+  margin-left: 100px;
+}
+
+#welcome-section {
+  background-color: rgb(189, 189, 190);
+  padding: 30px;
+  width: 100%;
+  height: 5em;
+}
+#projects h2{
+  font-size: 24px;
+  margin-left: 100px;
+  padding: 0px 400px 400px 0px;
+}
+#projects a{
+  font-size: 18px;
+  margin-left: 100px;
+  border-bottom: 1px solid blue;
+  color: blue;
+  padding: 0px;
+}
+#profile-link{
+  padding: 50px;
+  margin-left: 25%;
+}
+#profile-link button{
+  padding: 15px;
+  margin-left: 25%;
+  border: none;
+  background-color: rgb(164, 163, 163);
+  border-radius: 10px;
+  cursor: pointer;
+  color: blue;
+  font-family: "Josefin Sans", sans-serif;
+  font-size: 20px;
+}
+img{
+  height: 100%;
+  object-fit: cover;
+}
+@media (min-width: 480px) and (max-width:1200px){
+  body{
+    background-color: rgb(207, 203, 203);
+}
+}
```

Comment:
Be kind and follow `coding best practices` while coding to have clean and very standardized codes. 
Let me share with you some links which may be helpful for you while going through such practice in the feature.
- HTML & CSS best practices : https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md
- Watch this short video having the core subject on this topic then come up with the best code practices to enhance the understanding of the viewers. 
- Inside your CSS file make sure you test your code before submission: https://github.com/microverseinc/linters-config,
- Document your code in a professional way: https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/clean_repo.md
- Watch this short youtube video giving an overview of the entire of coding best practices. https://www.youtube.com/watch?v=rii6QRX8ZLs
Go ahead, you are almost done @KABIKA681 👏😊👏👍